Cargo.toml

```diff
@@ -1,17 +1,20 @@
 [package]
+authors = ["Richard Newman <rnewman@twinql.com>", "Nicholas Alexander <nalexander@mozilla.com>"]
 name = "mentat"
 version = "0.4.0"
-authors = ["Richard Newman <rnewman@twinql.com>", "Nicholas Alexander <nalexander@mozilla.com>"]
 
 [dependencies]
-rusqlite = "0.8.0"
 clap = "2.19.3"
+rusqlite = "0.8.0"
+slog = "1.4.0"
```

Comment:
For my future self: `slog` is licensed MPL2.  I don't know if APL can depend on MPL2!  (I know that MPL2 can depend on APL.) @rnewman, @joewalker do you know off the top?